tests/e2e/e2e_setup_test.go

```diff
@@ -621,86 +609,75 @@ func noRestart(config *docker.HostConfig) {
 }
 
 func (s *IntegrationTestSuite) writeGovProposals(c *chain) {
```

Comment:
Why comment this out instead of removing its usage?

Reply:
because it should be added back in some format rather than removed completely as other things that aren't going to make it into Rho with v0.45